socorro/cron/jobs/ftpscraper.py

```diff
@@ -313,11 +329,7 @@ def _scrape_json_releases_and_nightlies(
 
     def _insert_build(self, cursor, *args, **kwargs):
         self.config.logger.debug('adding %s', args)
-        if self.config.dry_run:
-            print "INSERT BUILD"
-            print args
-            print kwargs
-        else:
```

Comment:
It's output is no different from the line above. The logging. On stdout you get:

```
2017-08-11 11:23:11,393 DEBUG - SocorroAppBaseClass -  - MainThread - adding ('firefox', '49.0.1', 'win', u'20160922113459', 'release', None, u'mozilla-release', 'build3')
INSERT BUILD
('firefox', '49.0.1', 'win', u'20160922113459', 'release', None, u'mozilla-release', 'build3')
{'ignore_duplicates': True}
```
When I ran this a bunch of test to test I noticed the excess and thought it's no longer worth keeping.